s3tests_boto3/functional/__init__.py

```diff
@@ -244,6 +247,12 @@ def setup():
         config.storage_classes = ""
         pass
 
+    try:
+        config.lc_debug_interval = int(cfg.get('s3 main',"lc_debug_interval"))
+    except (configparser.NoSectionError, configparser.NoOptionError):
+        config.lc_debug_interval = 10
+        pass
```

Comment:
pass not needed if there is any statement on the branch?

Reply:
done